api/src/test/java/io/opentelemetry/trace/TraceIdTest.java

```diff
@@ -50,9 +50,13 @@ public void isValid() {
   }
 
   @Test
-  public void getLowerLong() {
-    assertThat(first.getLowerLong()).isEqualTo(0);
-    assertThat(second.getLowerLong()).isEqualTo(-0xFF00000000000000L);
+  public void testCorrectGetRandomTracePart() {
+    byte[] id = {
+      0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8,
+      0x9, 0xA, 0xB, 0xC, 0xD, 0xE, 0xF, 0x0
+    };
+    TraceId traceid = TraceId.fromBytes(id, 0);
+    assertThat(traceid.getTraceRandomPart()).isEqualTo(0x90A0B0C0D0E0F00L);
```

Comment:
It took me few minutes to check that this is the correct expectation, I would suggest to put a `0` in front of the `9` so having something like `0x090A0B0C0D0E0F00L` so I have 16 hex characters and also a easier way to map every 2 hex characters from here to the bytes.